docs/quick_start.md

```diff
@@ -0,0 +1,195 @@
+- Open an image
+- Image display adjustment
+- Manually label the cell
+- Get the cell area measurement
+- Next steps
+
+Along the way, you will see how to access napari functions from [Python code](https://napari.org/api/stable/index.html) and from GUI - though for different purposes, one method might be easier than another. This quick start guide will not cover ALL possible methods but only some ways to perform basic tasks. For the more complete guide, please visit [napari.org](https://napari.org/).
+
+You will also see some examples of plugins. The core napari viewer focuses on domain-agnostic functions such as layer controls. Analyses and domain specific functions, such as reading a special file format and image segmentation, live in the realm of [plugins](https://www.napari-hub.org/).    
+
++++
+
+### Installation
+
+- For those familiar with Python:
+
+    napari can be installed on most macOS, Linux, and Windows systems with Python 3.7, 3.8, and 3.9 using pip:
+
+```python
+pip install 'napari[all]'
+```
+
+- Or download the bundled app for simple installation:
+
+    [Linux installation](https://github.com/napari/napari/releases/download/v0.4.13/napari-0.4.13-Linux-x86_64.zip)<br>
+    [macOS installaion](https://github.com/napari/napari/releases/download/v0.4.13/napari-0.4.13-macOS-x86_64.zip)<br>
+    [Windows installation](https://github.com/napari/napari/releases/download/v0.4.13/napari-0.4.13-Windows-x86_64.zip)<br>
+
+    *Note: 0.4.13 bundled app is missing sample menu data ([issue 3968](https://github.com/napari/napari/issues/3968))*
+    
+    *Note: for the latest release, please visit [here](https://github.com/napari/napari/releases) and look for Assets.*
+
+If you run into any issue, please visit the more detailed [installation guide](https://napari.org/tutorials/fundamentals/installation.html), or report an issue on GitHub!
+
++++
+
+### Open napari
+
+napari can be opened by [multiple ways](https://napari.org/tutorials/fundamentals/getting_started.html), depending on how its used in your image analysis workflow. 
+
+Here we will be mainly focused on the GUI application.
+
+- From command line: 
+
+    Once installed, simply run    
+```python
+napari
+```
+
+- If you installed the bundled app:
+
+    Click on the app icon to open it.<br>
+    *Note: for mac users, it may require [security setting changes](https://napari.org/tutorials/fundamentals/installation.html).*
+
++++
+
+### Open an image
+
+napari natively supports tiff and many other formats supported by [skimage.io.imread](https://scikit-image.org/docs/dev/api/skimage.io.html) as input image file format.<br>
+Try with your own images or download [this ome tiff file](https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/00001_01.ome.tiff).
+
+Additional input file formats may be supported [by plugins](https://www.napari-hub.org/). 
+Try [napari-aicsimageio](https://www.napari-hub.org/plugins/napari-aicsimageio) if you have czi, lif, or nd2 files.
```

Comment:
This is a bit tricky! `aicsimageio` is *awesome* but just installing using the napari GUI will not provide czi, lif, or nd2 reading. It *will* enable out-of-memory reading though, so it's still sweet, but to get those formats you need to install extra things using pip. Further, in the case of CZI the key lib is compiled so it may not install on all platforms.

Reply:
@psobolewskiPhD is there a reader plugin(s) that you use regularly and think would be a good example instead of aicsimageio?

Reply:
AICSImageIO was one of the things that moved me to napari and python... 
It's the one I use—but it requires a bit extra on the install side.
There was some idea of changing the license on it so it would be able to bundle the proprietary types in the napari plugin for ease of (GUI) use. 
Perhaps @JacksonMaxfield can chime in?

Reply:
Hey all! I would _love_ to see this recommendation make it in personally. The holdup has honestly been me -- grad school and other projects have left me with little time :sweat_smile:

That said, iirc, we got approval from allen legal to move napari-aicsimageio over to GPL license and bundle all the various reader into the dependencies. So I will try to get to that this week and make a new release that fixes that. The recommendation here then would be: "napari-aicsimageio is good for lots of different file formats but comes with a GPL license"? Idk, it seems weird to include but hard to say how much of napari-aicsimageio is used for downstream processing / analysis and not only viewing. Like will napari-aicsimageio taint the rest of the workflow constructed by a scientist with a GPL license?

Reply:
The other option is that napari plugins can somehow communicate extra depends to the user. I've played around with ipython magics (https://github.com/napari/napari/discussions/3899#discussioncomment-1894365)
and `%pip install readlif` should JustWork in the napari console. the CZI one is compiled, so it's trickier, not sure about nd2...

Reply:
Update: I hope to have a napari-aicsimageio patch out next week. Sorry for the delay but we have actually had development progress hahaha

Reply:
Update: this should now be good to go. napari-aicsimageio v0.5.0 has been released that fixes a lot of stuff